cmd/ui-errors.go

```diff
@@ -47,6 +47,14 @@ var (
 		"MINIO_CACHE_EXPIRY: Valid cache expiry duration is in days.",
 	)
 
+	uiErrInvalidUsageCheckIntervalValue = newUIErrFn(
+		"Invalid usage check interval value",
+		"Please check the passed value",
+		`MINIO_USAGE_CHECK_INTERVAL: Valid usage check interval duration string is a signed sequence of decimal numbers,
+each with optional fraction and a unit suffix, such as "2h45m".
+Valid time units are "ns", "us", "ms", "s", "m", "h".`,
```

Comment:
@harshavardhana, the indentation is off on the last 2 lines of the help message.
```
ERROR Unable to parse MINIO_USAGE_CHECK_INTERVAL value (`abc`): Invalid usage check interval 
      value.
      > Please check the passed value.
      HELP:
        MINIO_USAGE_CHECK_INTERVAL: Valid usage check interval duration string is a signed sequence of
       decimal numbers,
      each with optional fraction and a unit suffix, such as "2h45m".
      Valid time units are "ns", "us", "ms", "s", "m", "h".

```